app/models/program.rb

```diff
@@ -199,4 +201,11 @@ def check_languages_format
     # We check if every language is a valid ISO 639-1 language
     errors.add(:languages, 'must be ISO 639-1 valid codes') unless languages_array.select{ |x| ISO_639.find(x).nil? }.empty?
   end
+
+  ##
+  # Check if schedule_cell_size is multiple of 60 minutes
+  #
+  def schedule_cell_size_multiple_60
```

Comment:
It should be a divisor of 60, not a multiple. Please change the name of the method too. :wink: